unit_tests/package.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
remove `SergiiMostepan/node-js/unit_tests/package.json`

move this test modules to `SergiiMostepan/node-js/package.json`

"mocha": "^8.1.1",
"sinon": "^9.0.2"
"should": "^13.2.3"